src/test/java/ioanandroflorea/tests/Tema8/pageObjects/DisableDynamicButtonsPage

```diff
@@ -0,0 +1,76 @@
+package ioanandroflorea.tests.Tema8.pageObjects;
```

Comment:
Please put it in a .java class file. I think you added it only as a simple file. Please also pay attention to the name of the class and file: DisabledDynamicButtonsPage vs DisableDynamicButtonsPage. They should be the same. Thanks!

Reply:
I realized I had an error here, but I didn't know where it came from. I hadn't looked carefully at the names of the classes.